backend/tests/test_contact_us.py

```diff
@@ -20,7 +20,7 @@ def test_contact_us_without_email(client, new_msg):
     """Test sending message with no email provided."""
     del new_msg["email"]
     rv = client.post('/send-email/', json=new_msg)
-    response = rv.get_json()
+    response = rv.get_json()['message']
```

Comment:
Two remarks here:
 * there's something essentially wrong when we're returning something like that:
```
{
    'message': {
        'content': {
            'code': 400,
            'message': 'Content of message is required'
        }
    }
}

```
as the body of a response. That need to be discussed so created separate ticket for that: https://github.com/CodeForPoznan/codeforpoznan.pl_v3/issues/132
 * if you have `response["content"]["message"]` at the bottom I don't see a lot of reason to do something like that: `response = rv.get_json()['message']`. Instead, you can have something like:
```
    response = rv.get_json()
    assert rv.status == "400 BAD REQUEST"
    assert response['message']["content"]["message"] == "Content of message is required"

```

Reply:
Also we could use here `asser rv.status_code == http.HTTPStatus.400_BAD_REQUEST`